conf/default.cfg

```diff
@@ -108,15 +108,21 @@ lang = en
 #options for testing the word similarity module
 [word_sim]
 definitions_binary = %(fourlangpath)s/data/machines/longman_firsts.pickle
+#definitions_binary = %(fourlangpath)s/data/machines/4lang.pickle
```

Comment:
why comment out a line in the default.cfg? It's point is to provide values for all parameters.